src/code/z_bgcheck.c

```diff
@@ -24,6 +24,14 @@
 #define COLPOLY_IGNORE_ENTITY (1 << 1)
 #define COLPOLY_IGNORE_PROJECTILES (1 << 2)
 
+// floor raycast flags (frcFlags)
+#define BGCHECK_FRC_0 0
+#define BGCHECK_FRC_1 (1 << 0)
+#define BGCHECK_FRC_2 (1 << 1)
+#define BGCHECK_FRC_4 (1 << 2)
+#define BGCHECK_FRC_8 (1 << 3)
+#define BGCHECK_FRC_10 (1 << 4)
```

Comment:
We've been using the bit for default flag names so I suggest doing the same here (ie. 1, 2, 3, 4, etc instead of 1, 2, 4, 8)